clients/drcachesim/tracer/raw2trace.h

```diff
@@ -958,6 +982,21 @@ template <typename T> class trace_converter_t {
         return "";
     }
 
+    std::string
+    process_memref(void *tls, trace_entry_t **buf_in, const instr_summary_t *instr,
+                   instr_summary_t::memref_summary_t memref, bool write,
+                   std::unordered_map<reg_id_t, addr_t> &reg_vals, uint64_t cur_modoffs,
+                   bool instrs_are_separate, OUT bool *reached_end_of_memrefs,
+                   OUT bool *interrupted)
+    {
+        std::string error = append_memref(tls, buf_in, instr, memref, write, reg_vals,
+                                          reached_end_of_memrefs);
+        if (!error.empty())
+            return error;
+        error = handle_kernel_interrupt_and_markers(
+            tls, buf_in, cur_modoffs, instr->length(), instrs_are_separate, interrupted);
+        return error;
+    }
```

Comment:
nit: Normally we separate functions with blank lines.  I thought clang-format enforced that?

Reply:
clang-format didn't throw any error. I fixed it though.